### DISCUSSION TITLE
Which is best?

### DISCUSSION THREAD

**Discussion by vishalakshisaraswat** [community/community#55450] [Programming Help]
### Select Topic Area

General

### Body

Hi,
I would like to know your views on which is the best full stack developer among :
 MERN, MEAN , PYTHON and JAVA full stack developer?

**Comment by thunderbolt990525** [comment]
✨ Hi : )

I think, ultimately, the best full-stack developer is one who possesses the necessary skills expertise to leverage the technology stack that best fits the project requirements and delivers a high-quality solution.

<img src="https://media.giphy.com/media/WUlplcMpOCEmTGBtBW/giphy.gif" width="30">
